src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -504,7 +510,20 @@ def get_non_host_dbs(dbs):
             return dbs
         else:
             return dbs[1:]
-        
+
+    @staticmethod
+    def get_start_idx_of_non_host(dbs):
```

Comment:
>get_start_idx_of_non_host [](start = 8, length = 25)

You can return a range since all use cases are in a range. You can further implement get_non_host_dbs by this function.

A better name will be `get_non_host_db_indexes`

Reply:
updated as per comment.